spec/spec_helper.rb

```diff
@@ -15,13 +24,14 @@
         app: "#{ENV['app']}",
         deviceOrientation: 'portrait',
         name: example.full_description,
-        appiumVersion: '1.4.11',
+        appiumVersion: '1.5.2',
```

Comment:
We seem to have having issues with 1.5.x at the moment using 1.4.16 seems to be safest.